index.js

```diff
@@ -0,0 +1,33 @@
+const express = require('express');
+const port = process.env.PORT || 4000;
+
+const eventsRouter = require('./events/router')
+const userRouter = require('./users/router')
+const cuttingRouter = require('./cuttings/router')
+const commentRouter = require('./comments/router')
+const pastEventRouter = require('./events-photo-album/router')
+const eventPhotoRouter = require('./events-photos/router')
+
+const cors = require('cors')
+const bodyParser = require('body-parser')
+
+const app = express();
+
+const corsMiddleware = cors()
+app.use(corsMiddleware)
+
+const parserMiddleware = bodyParser.json()
```

Comment:
use express own parser `const parserMiddleware = express.json()` so you use one less dependency

Reply:
Oh yeah! That's right